express/blocks/columns/columns.css

```diff
@@ -120,4 +140,40 @@ main .columns .social.icon {
   main .columns.fullsize + h2 {
     margin-top: 0;
   }
+}
+
+@media (max-width: 899px) {
+  main .columns .hero-animation-overlay {
+    order: -1;
+  }
+
+  main .new-template-list-sixcols-container > div > .button-container {
+    position: -webkit-sticky;
+    position: sticky;
+    bottom: 40px;
+  }
+
+  main .new-template-list-sixcols-container > div > .button-container a.button {
+    font-size: var(--body-font-size-l);
+    margin-top: 40px;
+    padding: 14px 30px;
+    border-radius: 30px;
+    font-weight: 400;
+    animation-name: floating-up-animation;
+    animation-duration: 1s;
+  }
+
+  main .new-template-list-sixcols-container > div > div .button-container {
+    display: none;
```

Comment:
In the beginning, the first button (inside the marquee) must not be hidden, but appear between the heading and the first text paragraph. Only when scrolled out of view (you could use a fixed `scrollY` value from the top), it should start floating at the bottom.

My guess (to be confirmed with design) is that all other CTAs should be hidden while the main CTA is floating...